app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -134,8 +141,7 @@ class AdministratorControlsActivity :
     if (fragment is ProfileEditFragment) {
       administratorControlsActivityPresenter.handleOnBackPressed()
     } else {
-      @Suppress("DEPRECATION") // TODO(#5404): Migrate to a back pressed dispatcher.
-      super.onBackPressed()
+      finish()
```

Comment:
When you finish the current activity, does it go back to the previous expected activity?